ckanext/data_qld/helpers.py

```diff
@@ -153,7 +153,11 @@ def is_delete_request():
 
 
 def is_api_request():
-    return get_request_action() == 'action' or '/action/' in get_request_path()
+    action = get_request_action()
+    path = get_request_path()
+    return (action == 'action' or '/action/' in path) \
```

Comment:
test plz.

Reply:
It's difficult to test the resource reorder page, since it relies on a JavaScript drag-and-drop, which then assembles a custom POST request.

Reply:
was meaning via unit testing (all the 'combinations' this logic is now doing, tis is not a simple thing anymore)

having a gui test would be great too, do review ckan/ckan codebase to see if they do indeed have one. if not. having a bahave extension that helps us do this would be an awesome portfolio item for the developer who writes it.

Reply:
I've split this so we're no longer making this function so complex.